### PR TITLE
Workaround for bug on odb call

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: modules/server_new/src/clue/scala/observe/common/ObsQueriesGQL.scala [hunk 1]
```diff
@@ -363,10 +363,7 @@ object ObsQueriesGQL:
               label
               observation {
                 label
-                program {
```
Comment (cquiroz) [inline modules/server_new/src/clue/scala/observe/common/ObsQueriesGQL.scala:366]: This call is throwing an assertion on the odb but I'm unable to reproduce it in a test

File: modules/server_new/src/main/scala/observe/server/keywords/StandardHeader.scala [hunk 2]
```diff
@@ -73,7 +73,7 @@ class StandardHeader[F[_]: Sync: Logger](
       buildString(obsObject, KeywordName.OBJECT),
       buildString(obsReader.obsType, KeywordName.OBSTYPE),
       buildString(obsReader.obsClass, KeywordName.OBSCLASS),
-      buildString(dataset.map(_.observation.program.label.label).getOrElse("").pure[F],
+      buildString(dataset.map(_.observation.label.programReference.label).getOrElse("").pure[F],
```
Comment (rpiaggio) [inline modules/server_new/src/main/scala/observe/server/keywords/StandardHeader.scala:76]: Does this return the same value? Maybe we can leave it like this.

**Review by rpiaggio — APPROVED** [review]
LGTM

File: modules/server_new/src/main/scala/observe/server/keywords/StandardHeader.scala (hunk 2 above)
Comment (cquiroz) [reply modules/server_new/src/main/scala/observe/server/keywords/StandardHeader.scala:76]: it should be the same

**Pull request merged**